packages/utils/src/screen.js

```diff
@@ -3,5 +3,7 @@ import { PixelRatio } from "react-native";
 import { acceptedWidths, normaliseWidth, screenWidth } from "./screen.base.js";
 
 export { acceptedWidths, normaliseWidth, screenWidth };
-export const screenWidthInPixels = () =>
-  PixelRatio.getPixelSizeForLayoutSize(screenWidth());
+
+export const convertToPixels = px => PixelRatio.getPixelSizeForLayoutSize(px);
```

Comment:
We are passing in points, not pixels, so I suggest something like this to avoid confusion:
```suggestion
export const convertToPixels = points => PixelRatio.getPixelSizeForLayoutSize(points);
```